cfpb/capital-framework
# cf-notifications: Adjust message/explanation and links margins

## Changes

- Adjust margins of links in notifications.

## Testing

1. Pull branch down.
3. Ensure you're in the docs folder with `cd docs`
2. Run `gulp build && gulp docs && bundle exec jekyll serve` (run `gem install bundler && bundle` if jekyll is not installed.)

## Screenshots

<img width="567" alt="Screen Shot 2019-04-23 at 10 32 30 AM" src="https://user-images.githubusercontent.com/704760/56590109-7d2b0200-65b4-11e9-8184-6e776c1e7ef9.png">

---

<img width="678" alt="Screen Shot 2019-04-23 at 10 32 42 AM" src="https://user-images.githubusercontent.com/704760/56590110-7d2b0200-65b4-11e9-9148-a633ffcabae4.png">

---
The below are what one link would look like, but this is just for show and docs have two links only.
---

<img width="556" alt="Screen Shot 2019-04-23 at 10 32 04 AM" src="https://user-images.githubusercontent.com/704760/56590108-7d2b0200-65b4-11e9-8497-dcfe3d9c5c0a.png">

---

<img width="613" alt="Screen Shot 2019-04-23 at 10 31 52 AM" src="https://user-images.githubusercontent.com/704760/56590107-7d2b0200-65b4-11e9-9f69-37ae70e4e3c7.png">
